### PR TITLE
Add pages production concourse jobs and cg vars file

### PR DESCRIPTION
**Pull request by apburnes**
## Changes proposed in this pull request:
- Add pages production depoloyments to Concourse CI
- Add pages .cloudgov deployment vars file

## security considerations
Needed for Pages SCR migration

### CODE REVIEW

File: ci/pipeline.yml [hunk 1]
```diff
@@ -190,9 +311,26 @@ resources:
     type: git
     icon: github
     source:
-      uri: https://github.com/18F/federalist-proxy
+      uri: ((proxy-repository-url))
```
Comment (apburnes) [inline ci/pipeline.yml:314]: Thoughts on using env variable so we do not need to make another commit when migrating from 18F to cloud-gov org?

File: ci/pipeline.yml [hunk 2]
```diff
@@ -190,9 +311,26 @@ resources:
...
+    icon: github
+    source:
+      uri: ((proxy-repository-url))
+      branch: main
+
+  - name: pr
+    type: pull-request
+    check_every: 1m
+    source:
+      repository: 18F/federalist-proxy
```
Comment (apburnes) [inline ci/pipeline.yml:328]: The GH PR resource only needs the `org/repo` path so do we hardcode or change the `((proxy-repository-url))` variable and make it compound like `((proxy-git-base-url))/((proxy-repository-path))`

File: ci/pipeline.yml [hunk 3]
```diff
@@ -9,6 +9,13 @@ staging-cf: &staging-cf
   CF_ORG: gsa-18f-federalist
   CF_SPACE: staging
 
+staging-cf: &production-cf
```
Comment (hursey013) [inline ci/pipeline.yml:12]:
```suggestion
production-cf: &production-cf
```

File: ci/pipeline.yml (hunk 1 above)
Comment (hursey013) [reply ci/pipeline.yml:314]: Nice, good thinking!

File: ci/pipeline.yml (hunk 2 above)
Comment (hursey013) [reply ci/pipeline.yml:328]: Hopefully we don't do another org change soon, but I'd say if it's not too much effort this would be the way to go

File: ci/pipeline.yml [hunk 4]
```diff
@@ -240,4 +378,36 @@ resource_types:
...
\ No newline at end of file
+      repository: cfcommunity/slack-notification-resource
+
+  - name: pull-request
+    type: docker-image
+    source:
+      repository: teliaoss/github-pr-resource
+
+############################
+# Groups - Use to better visualize and track jobs in CI
```
Comment (hursey013) [inline ci/pipeline.yml:389]: Haven't seen `groups` before, do we use these elsewhere?

**Review by hursey013 — APPROVED** [review]
👍

File: ci/pipeline.yml (hunk 4 above)
Comment (apburnes) [reply ci/pipeline.yml:389]: Not in Pages but we do in cloud-gov. We don't have too complex of pipelines but it maybe nice to visualize the deployments while we straddle Pages and Federalist.

File: ci/pipeline.yml (hunk 2 above)
Comment (apburnes) [reply ci/pipeline.yml:328]: Sounds good. _Note: There may be a point in the future where cloud.gov switches git servers._
